elastic/integrations
# Memcached_Integration: Initial Integration

## What does this PR do?

This PR enables having Memcached as an integration.

## Checklist



## How to test this PR locally
1. Run memcached on local system or as a docker image.
2. Setup the ELK stack 
3. Enable the integration on Kibana
4. You should see the metrics being received.
5. The dashboard would be populated as well

## Related issues

- Closes https://github.com/elastic/integrations/issues/372

## Screenshots
<img width="2534" alt="Memcached_Metrics_Overview_Dashboard_1" src="https://user-images.githubusercontent.com/102962586/175879647-a2622f0d-895c-4e1c-8922-f6fd71d8c9eb.png">

<img width="2541" alt="Memcached_Metrics_Overview_Dashboard_2" src="https://user-images.githubusercontent.com/102962586/175879669-e15e43d2-0096-4a90-85d1-120bc4a72be7.png">
